spec/stud/interval_spec.rb

```diff
@@ -38,5 +38,21 @@
 
       expect(counter).to eq(1)
     end
+
+    it 'should be able to wake up from an sleep' do
+      counter = 0
+
+      task = Stud::Task.new do
+        Stud.interval(2) do
+          counter += 1
+        end
+      end
+      task.stop!
```

Comment:
Any specific reason that `#stop!` doesn't implicitly call `#wakeup?
We already have the `!` in the signature, so I would expect it to stop right away?